lib/travis/build/script/smalltalk.rb

```diff
@@ -1,39 +1,69 @@
+# Copyright (c) 2015-2016 Software Architecture Group (Hasso Plattner Institute)
+# Copyright (c) 2015-2016 Fabio Niephaus, Google Inc.
+
 module Travis
   module Build
     class Script
       class Smalltalk < Script
-        DEFAULTS = {}
+        DEFAULTS = {
+          gemstone_hostname: 'travis.dev',
```

Comment:
@dalehenrich works a lot with GemStone and he has always used that hostname.
AFAIK, it only needs to be a predictable hostname, because Travis builds usually have a randomized one.